Extensions/Leaderboards/JsExtension.js

```diff
@@ -59,6 +59,11 @@ module.exports = {
         false
       )
       .addParameter('string', _('Name to register for the player'), '', false)
+      .setParameterLongDescription(
+        _(
+          'Let this empty to let the leaderboard auto-generate a player name.'
```

Comment:
Should it mention that it will look like Player23464?

Reply:
Will do